marshal/oracle/backup.go

```diff
@@ -46,6 +46,7 @@ func Backups(cmdOutput []byte) []model.OracleDatabaseBackup {
 			avgBckSize, err := strconv.ParseFloat(splitted[3], 64)
 			if err != nil {
 				log.Printf("%v\n", err)
+				return backups
```

Comment:
This is wrong: if you get an error you can't stop and return with only the values find until now.
We must manage the errors and return them with the values:
collect errors in a `multierror` and if any return that instead of the backups!

Reply:
Take a look at https://github.com/ercole-io/ercole-agent/blob/2ac5331a62d4ed10ead87125728b36623d206558/marshal/oracle/addm.go as an example